lib/WebService/Fastly.pm

```diff
@@ -0,0 +1,41 @@
+package WebService::Fastly;
+
+use strict;
+use 5.008_005;
+our $VERSION = '2.01';
+
+1;
+__END__
+
+=encoding utf-8
+
+=head1 NAME
+
+WebService::Fastly - an interface to most facets of the [Fastly API](https://developer.fastly.com/reference/api).
+
+=head1 SYNOPSIS
+
+  use WebService::Fastly;
```

Comment:
I feel like we need a better synopsis than this.

Maybe

1. Login
2. Get a list of services
3. Clone a service
4. Change something (add Header and/or Domain)
5. Deploy

And maybe something like adding something to a Versionless Dictionary to show the different mode of interaction.

Reply:
@simonwistow thanks for the comments.  I'll make a new ticket for this and get these added in the next (2.03) release.